### DISCUSSION TITLE
Unable to login for the first time on JTL Reporter

### DISCUSSION THREAD

**Discussion by gelsalgado** [ludeknovy/jtl-reporter#184] [Q&A] [answered]
I am trying to get JTL reporter running in Ubuntu 18.04. I have finished running docker-compose and trying to login to the application for the first time but I am getting an error.
I tried using the default admin credentials from the site or try using a new username/password but in both instances, it throws an error. Is there an additional step I am missing? Please help.
![Screenshot from 2022-09-09 12-38-39](https://user-images.githubusercontent.com/28947736/189267024-558c3f45-3e11-4d92-b9f7-e4b13ea64f43.png)
![Screenshot from 2022-09-09 12-40-57](https://user-images.githubusercontent.com/28947736/189267166-2870c731-de23-4291-9bda-2e466ec553ba.png)

**Comment by gelsalgado** [comment, marked as answer]
I was able to make it work by killing all the docker images and running docker-compose once more.

**Reply by ludeknovy** [reply]
Hi @gelsalgado. Glad you made it work :)
